en/docs/administer/logging-and-monitoring/logging/setting-up-logging-per-api.md

```diff
@@ -0,0 +1,62 @@
+---
+template: templates/swagger.html
+---
+# Setting up logging per API in API Manager
+
+!!! note
+    Enabling logs per API is only available after the WUM update with the timestamp "1600491803101"
+
+In a production envrionment, having the feasbility to monitor HTTP message flow for a given API is important in identifying issues in the HTTP message flow.
+
+This feature enables API requests logging at four levels.
+
+
+1. Request coming to the gateway from client (request-in>>)
+2. Request leaving the gateway to backend (request-out<<)
+3. Request coming to the gateway from backend (response-in>>)
+4. Request leaving to the client from gateway (response-out<<)
+
+Depending on the nature of the request, you can chose only to log headers, body or both.
+
+A REST API secured with basic authentication is provided to enable/disable API logs at the run time. Only a user with admin role has priviledges to invoke the API.
+
+
+
+
+??? info "Click here to see how to try out the APIs"
+    Do the following to try out the REST APIs with your local instance of WSO2 API Manager.
+
+       1. Expand the relevant API operation and click the **Try It Out** button.
```

Comment:
We need to provide steps to find this, to guide a user from the UI.

Reply:
Swagger UI is there to populate the documentation from the given swagger definition.